src/ch/ehi/umleditor/application/ModelDefDialog.java

```diff
@@ -1402,6 +1403,30 @@ private DescriptionPanel getPnlDescription() {
 	}
 	return ivjPnlImport;
 }
+/**
+ * Return the PnlMetaAttributes property value.
+ * @return ch.ehi.umleditor.application.MetaAttributePanel
+ */
+/* WARNING: THIS METHOD WILL BE REGENERATED. */
+private MetaAttributePanel getPnlMetaAttributes() {
+	if (ivjPnlMetaAttributes == null) {
+		try {
+			ivjPnlMetaAttributes = new MetaAttributePanel(this, Arrays.asList(
+					ch.interlis.ili2c.metamodel.Ili2cMetaAttrs.ILIMODELSXML_TECHNICAL_CONTACT,
+					ch.interlis.ili2c.metamodel.Ili2cMetaAttrs.ILIMODELSXML_FURTHER_INFORMATION,
+					ch.interlis.ili2c.metamodel.Ili2cMetaAttrs.ILIMODELSXML_ID_GEO_IV));
+
```

Comment:
Q: Wird die `ignoredMetaAttributes`-Liste regeneriert wenn der UI-Editor verwendet wird?

Reply:
Bei jedem Aufruf von `Arrays.asList()` wird eine neue Listen-Instanz erstellt. `ivjPnlMetaAttributes` wird allerdings nur einmal pro ModelDefDialog instantiiert.

Reply:
Hier hast du wohl meine Frage falsch verstanden: Die Methode hat den Kommentar /`* WARNING: THIS METHOD WILL BE REGENERATED. */` Wenn hier diese Generierung stattfindet, verlieren wir dann den Filter da es nicht im User-Code liegt?